app/services/romm_auto.py

```
@@ -0,0 +1,644 @@
+            considered += 1
+            if not _passes_filters(rom, rule):
+                result["skipped_filtered"] += 1
+                continue
+            path = romm_client.local_path(rom)
+            if not path or not is_within_configured_volumes(path):
+                continue
+            if not await run_in_threadpool(tool.converts_path, path):
+                result["skipped_unconvertible"] += 1
+                continue
+            try:
+                resolved = str(Path(path).expanduser().resolve(strict=False))
```

Ruff (0.16.1)

[warning] Async functions should not use pathlib.Path methods, use trio.Path or anyio.path

(ASYNC240)